utils/7z/Makefile

```diff
@@ -0,0 +1,68 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=7z
+PKG_VERSION:=22.01
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)$(subst .,,$(PKG_VERSION))-src.tar.xz
+PKG_SOURCE_URL:=https://7-zip.org/a/
+PKG_HASH:=393098730c70042392af808917e765945dc2437dee7aae3cfcc4966eb920fbc5
+
+PKG_MAINTAINER:=Tianling Shen <cnsztl@immortalwrt.org>
+PKG_LICENSE:=LGPL-2.1-or-later BSD-3-Clause
+PKG_LICENSE_FILES:=DOC/License.txt DOC/copying.txt
+PKG_CPE_ID:=cpe:/a:7-zip:7-zip
+
+HOST_BUILD_PARALLEL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
Why?

Reply:
cloudreve needs it to create zip archive.

Reply:
This is a host package already provided by tools/7zip.

Reply:
Yep, but it's 7zr, which can only handle .7z archives.

Reply:
If the compilation options for tools/7zip are inadequate, submit a patch to fix those.

Reply:
Wait a minute, you're creating a zip file. Why not use libdeflate instead?

Reply:
> Wait a minute, you're creating a zip file. Why not use libdeflate instead?

If I understand correctly libdeflate can only create gzip archive, it's not suitable here.

And I see zip is already in tools https://github.com/openwrt/openwrt/tree/main/tools/zip, I must have missed something

Reply:
Ah right. libdeflate is only for creating gzip files.